easybuild/framework/easyconfig/default.py

```diff
@@ -155,12 +155,13 @@
     'exts_list': [[], 'List with extensions added to the base installation', EXTENSIONS],
 
     # MODULES easyconfig parameters
+    'modaliases': [{}, "Aliases to be defined in module file", MODULES],
     'modextrapaths': [{}, "Extra paths to be prepended in module file", MODULES],
     'modextravars': [{}, "Extra environment variables to be added to module file", MODULES],
     'modloadmsg': [{}, "Message that should be printed when generated module is loaded", MODULES],
     'modluafooter': ["", "Footer to include in generated module file (Lua syntax)", MODULES],
+    'modname': [None, "Module name to use (rather than using software name", MODULES],
```

Comment:
Sounds good

On Tue, Sep 15, 2015, 3:53 PM Kenneth Hoste <notifications@github.com>
wrote:

> In easybuild/framework/easyconfig/default.py
> <https://github.com/hpcugent/easybuild-framework/pull/1389#discussion_r39556436>
> :
>
> >      'modextrapaths': [{}, "Extra paths to be prepended in module file", MODULES],
> >      'modextravars': [{}, "Extra environment variables to be added to module file", MODULES],
> >      'modloadmsg': [{}, "Message that should be printed when generated module is loaded", MODULES],
> >      'modluafooter': ["", "Footer to include in generated module file (Lua syntax)", MODULES],
> > +    'modname': [None, "Module name to use (rather than using software name", MODULES],
>
> I like modaltsoftname: descriptive and long enough so you'll think twice
> before using it. @rjeschmi <https://github.com/rjeschmi>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcugent/easybuild-framework/pull/1389/files#r39556436>
> .
>